tests/integration/rdb.tcl

```diff
@@ -198,3 +198,73 @@ test {client freed during loading} {
         exec kill [srv 0 pid]
     }
 }
+
+start_server {overrides {save ""}} {
+    test {Test child sending COW info} {
+        r config set rdb-key-save-delay 200
```

Comment:
let's put another comment with some raw description of the test, something in the lines of:
- create 10k keys of 4k each
- using a 200us sleep the bgsave is empirically taking about 10 seconds.
- we need it to take more than some 5 seconds since redis only report COW once a second.
- the test runs a few cycles in which it overrides a key, and waits to see that change reported in INFO